packages/glibc_lib237.rb

```diff
@@ -42,10 +41,42 @@ def self.install
       FileUtils.install @filename_target, @destpath
     end
     # Symlinks to system libraries.
-    %w[libanl.so.1 libc_malloc_debug.so.0 libc.so.6 libdl.so.2 libm.so.6
-       libmvec.so.1 libnss_dns.so.2 libnss_files.so.2 libpthread.so.0
-       libresolv.so.2 librt.so.1 libthread_db.so.1 libutil.so.1].each do |lib|
-      FileUtils.ln_sf "/#{ARCH_LIB}/#{lib}", "#{CREW_DEST_LIB_PREFIX}/#{lib}"
+    @glibc_libs = %w[libanl.so.1 libc_malloc_debug.so.0 libc.so.6 libdl.so.2 libm.so.6
+                     libmvec.so.1 libnss_dns.so.2 libnss_files.so.2 libpthread.so.0
+                     libresolv.so.2 librt.so.1 libthread_db.so.1 libutil.so.1]
+    @glibc_libs -= ['libc.so.6'] if Kernel.system("patchelf --print-needed #{File.join(CREW_LIB_PREFIX, 'libc.so.6')} | grep -q libC.so.6")
+    @glibc_libs -= ['libm.so.6'] if Kernel.system("patchelf --print-needed #{File.join(CREW_LIB_PREFIX, 'libm.so.6')} | grep -q libC.so.6")
+
+    @glibc_libs.each do |lib|
+      if ARCH == 'x86_64'
+        puts "skipping symlink of system/#{ARCH_LIB}/#{lib} into #{CREW_DEST_LIB_PREFIX}/#{lib}...".orange
+      else
+        FileUtils.ln_sf "/#{ARCH_LIB}/#{lib}", "#{CREW_DEST_LIB_PREFIX}/#{lib}"
+      end
+    end
+  end
+
+  def self.postinstall
+    if (ARCH == 'x86_64') && (LIBC_VERSION.to_f >= 2.35)
```

Comment:
```suggestion
    if (ARCH == 'x86_64') && (LIBC_VERSION >= '2.35')
```

from my testing in irb, no need to convert to a float here.

Reply:
Looks like the conversion is necessary:

```
Performing post-install for glibc_lib237...
/usr/local/lib/crew/packages/glibc_lib237.rb:60:in `>=': comparison of String with 2.35 failed (ArgumentError)
```